gui/wxpython/location_wizard/wizard.py

```diff
@@ -1630,59 +1658,37 @@ def OnText(self, event):
             self.epsgcode = None  # not found
             if nextButton.IsEnabled():
                 nextButton.Enable(False)
-            self.epsgdesc = self.epsgparams = ''
-
-    def OnSearch(self, event):
+            self.epsgdesc = self.epsgparams = ''           
+            
         value = self.searchb.GetValue()
+        self.tlink.SetURL(str("https://epsg.io/?q={0}".format(value)))
+        self.tlink.SetLabel(str("https://epsg.io/?q={0}".format(value)))
```

Comment:
remove SetLabel, we want the label to be the same all the time, just the link changes